backend/app.py

```diff
@@ -77,6 +78,16 @@ def parse_user():
 	e = request.args.get('e')
 	return parseFile(url, b, e, request.args)
 
+@app.route('/api/volunteer/csv/', methods = ['GET'])
+@auth.login_required
+def build_csv():
+	try:
```

Comment:
What do you think about https://stackoverflow.com/questions/26997679/writing-a-csv-from-flask-framework ?

Reply:
Here is a piece of discussion with Ion:

_headers: {
    'Content-Type': 'text/csv'
  },
responseType: 'blob'
7:17
Asa mia zis o persoana pe frontend_

As I did understand on frontend side the file should not be an attachment.